data/schema-config.yaml

```diff
@@ -1161,6 +1161,15 @@ complex:
   label_in_input: complex
   db_collection_name: complexes
   db_collection_per_chromosome: false
+  db_indexes:
```

Comment:
should also add persistent index for complex_source and evidence_code

Reply:
they are not being used by any of the endpoints as search parameters

Reply:
Oh, I though filtering by complex_source and evidence_code will need those two fields to be indexed?

Reply:
I think we only need to return them